src/components/Common/TaxInputs/TaxInputs.tsx

```diff
@@ -96,7 +134,9 @@ export function RadioInput({ question, requirement }: EmpQ | CompR) {
     <RadioGroupField
       name={key}
       //File new hire report setting cannot be changed after it has been configured.
-      isDisabled={key.includes('fileNewHireReport') ? (value === undefined ? false : true) : false}
+      isDisabled={
+        key.includes('fileNewHireReport') ? (value === undefined ? false : true) : isDisabled
```

Comment:
This predicate `key.includes('fileNewHireReport`')` pops up a lot. What do you think about moving it to its own function?